diagnostics/model_test/README.md

```diff
@@ -1,21 +1,24 @@
-# Compare model output between Tensorflow and NGraph
+# Compare model output between two different backends
 
-### This model_test tool will run the model inference seperately on TF and NGraph, and the desired output from TF and NGraph should match given the same inputs. It can be used as a debugging tool, and also a verification that NGraph produces the same output as Tensorflow. 
+### This model_test tool will run the model inference seperately on two specified backends(e.g.Tensorflow and nGraph) in json file and the desired outputs from each backend should match given the same inputs. It can be used as a debugging tool for layer by layer comparison, and also a verification that nGraph produces the same output as Tensorflow. 
 
 # Required files to use the tool:
-* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json```. You can start with the ```template.json``` and modify it to match your model
+* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json```.
 * A tensorflow frozen graph: A model frozen graph(.pb file) with trained weights and model architecture
 
-## To prepare the required json file:"
+## To prepare the required json file:
+* Specify the ```reference_backend``` and ```testing_backend```. For tensolrflow on CPU, use 'CPU' and for nGraph, use 'NGRAPH_[desired backend name]' (e.g. Use 'NGAPH_CPU' for nGraph on CPU)
+* Specify ```quantized_mode``` as 1 if you use quantization. Otherwise set as 0 for float mode
 * You will need the names of the input/output tensors of the model. Currently we are supporting
-multiple input tensors and one output tensor. Put the input tensor names as a list in the ```input_tensor_name``` field of the json file, and the output tensor name as a string in the ```output_tensor_name``` field of the json file
+multiple input tensors and output tensors. Put the input tensor names as a list in the ```input_tensor_name``` field of the json file, and the output tensor name as a list in the ```output_tensor_name``` field of the json file. If no outputs are specified in the ```output_tensor_name```, then it will compare all output tensors
 * You will need the input dimensions for all the input tensors provided. Put the dimensions information as a list in the ```input_dimension``` field of the json file, and the corresponding order of ```input_tensor_name``` list should match the ```input_dimension``` list. Therfore, the length of ```input_tensor_name``` list should match the length of ```input_dimension``` list
 * Specify the the location of the frozen graph in the ```frozen_graph_location``` of the json file
 * Specify the ```batch_size``` field in the json file to the desired batch size for inference
 * Specify the tolerance between the TF and NGraph outputs at ```l1_norm_threshold```, ```l2_norm_threshold``` and ```inf_norm_threshold``` in the json file 
+* Specify the ```random_val_range``` used for the test
```

Comment:
Maybe change `used for the test` to
`used to generate the input within 0 to random_val_range`

Reply:
@mingshan-wang 
Yes, for now it is ok to specify only the max as an input, but generally we can support min and max as inputs, which we can do in a separate PR. We can provide defaults of 0 and 1 for min and max if nothing is specified